test/e2e/runner/cli_provisioner.go

```diff
@@ -283,25 +278,6 @@ func (cli *CLIProvisioner) waitForNodes() error {
 		}
 	}
 
-	if cli.Config.IsDCOS() {
```

Comment:
Shouldn't this section stay in acs-engine for backward compatibility? We can remove it aks-engine of course.

Reply:
no